outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "webapp_url" {
+  value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+
+output "cd_webhook" {
+  value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
```

Comment:
Should be marked as `sensitive = true` so it doesn't show up in CI/CD logs

Reply:
Done.